python/ray/serve/tests/test_standalone.py

```diff
@@ -508,5 +508,47 @@ def crash():
     crash()
 
 
+def test_output():
+    driver_script = """
+from fastapi import FastAPI
+import requests
+
+from ray import serve
+
+app = FastAPI()
+serve.start()
+
+@serve.deployment
+@serve.ingress(app)
+class A:
+    @app.get("/")
+    def say_hi(self):
+        return "hello"
+
+A.deploy()
+
+resp = requests.get("http://localhost:8000/A")
+resp.raise_for_status()
+
+A.delete()
+serve.shutdown()
```

Comment:
Does it make sense to add a short sleep after `shutdown`?  Sometimes when I run scripts I notice that messages come after the script exits, maybe the sleep would help capture these.